cmd/nerdctl/compose_run_linux_test.go

```diff
@@ -51,6 +51,7 @@ services:
 	t.Logf("projectName=%q", projectName)
 	defer base.ComposeCmd("-f", comp.YAMLFullPath(), "down", "-v").Run()
 
+	defer base.Cmd("rm", "-f", "-v", containerName).Run()
```

Comment:
Isn't `defer base.ComposeCmd("-f", comp.YAMLFullPath(), "down", "-v").Run()` enough?

Reply:
It didn't work in docker (compose), e.g., containers from `docker compose run` (i.e., not from `docker compose up`) won't be covered by `docker compose down`:

```shell
$ cat docker-compose.yaml
version: '3.1'

services:
  svc0:
    image: nginx
  svc1:
    image: rabbitmq

# docker compose down cannot down container from docker compose run
$ docker compose run -d svc0
[+] Running 1/0
 ✔ Network tmp_default  Created                                                                                                                                                                                                                                                                                                       0.0s
103f148596f2d629f2b875d83da6ac5ea3682ad85b8c61d58e73ddc7931f3b11
$ docker compose down -v
[+] Running 1/0
 ✘ Network tmp_default  Error                                                                                                                                                                                                                                                                                                         0.0s
failed to remove network tmp_default: Error response from daemon: error while removing network: network tmp_default id 3795360bd3e1e9a11def2a76f140597f328009fb101bd7d67490bdeebb8b54f9 has active endpoints
$ docker ps
CONTAINER ID   IMAGE     COMMAND                  CREATED          STATUS          PORTS     NAMES
103f148596f2   nginx     "/docker-entrypoint.…"   24 seconds ago   Up 23 seconds   80/tcp    tmp-svc0-run-db8a05f1f3e6

# works fine if from docker compose up
$ docker compose up -d
[+] Running 2/2
 ✔ Container tmp-svc1-1  Started                                                                                                                                                                                                                                                                                                      0.2s
 ✔ Container tmp-svc0-1  Started                                                                                                                                                                                                                                                                                                      0.3s
$ docker compose down -v
[+] Running 3/2
 ✔ Container tmp-svc0-1  Removed                                                                                                                                                                                                                                                                                                      0.1s
 ✔ Container tmp-svc1-1  Removed                                                                                                                                                                                                                                                                                                      6.3s
 ✔ Network tmp_default   Removed
```